tests/test_rpc.py

```diff
@@ -22,6 +22,7 @@
 
 def test_getinfo(lianad):
     res = lianad.rpc.getinfo()
+    assert 'timestamp' in res.keys()
```

Comment:
nit: could've sanity checked the value. For instance that it's after now - 1 day and lower or equal to now.